feat: Add finish timestamp for every browser

theghostbel opened the pull request:
Being applied we can see in console time (HH:MM:SS) when run completed. This helps in situations when you work for a while over some file and then look into console and don't know if current results are the freshest? You throw a look on clocks and realise - hey! it's five minutes old! I've forgot to click "CTRL+S"

```
Firefox 34.0.0 (Windows 7): Executed 98 of 98 (2 FAILED) (8.926 secs / 8.867 secs) finished at 01:16:27

Dignifiedquire commented:
Thanks, can you please change the commit message [following this git commit message convention](http://karma-runner.github.io/0.12/dev/git-commit-msg.html).

maksimr commented on lib/reporters/base.js:
```diff
@@ -38,6 +38,13 @@ var BaseReporter = function(formatError, reportSlow, adapter) {
+      var date = new Date();
+      var hours = date.getHours() < 10 ? '0' + date.getHours() : date.getHours();
+      var minutes = date.getMinutes() < 10 ? '0' + date.getMinutes() : date.getMinutes();
+      var seconds = date.getSeconds() < 10 ? '0' + date.getSeconds() : date.getSeconds();
```
@theghostbel we can improve code little:

```js
var finishDate = new Date();
var formatTime = function(date) { return time < 10 ? '0' + time : time};
msg += util.format(' finished at %s:%s:%s', formatTime(formatTime.getHours()), formatTime(finishDate.getMinutes()), formatTime(finishDate.getSeconds()));
```

theghostbel replied:
agreed

Dignifiedquire commented:
@theghostbel could you fix the code formatting issues that came up on travis please?

theghostbel commented:
@Dignifiedquire fixed, thanks for pointing!

Dignifiedquire commented:
Now all that's missing is to squash it into on commit then it's good to go from my side.

theghostbel commented:
Squashed commits into one. For those who is searching "how to" squash commits for github`s pull request into one I'd recommend this [article](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit). Worked like a charm after I switched to simple push mechanism for forced pushes:
````
$ git config --global push.default simple
$ git push -f
````

@Dignifiedquire, how to be when one of builds failed? :( Looks like it was kinda test build env ("iojs-1")

Dignifiedquire commented:
@theghostbel I've retriggered it lets see with what it comes back

Dignifiedquire commented:
@theghostbel could you rebase onto the latest master please?

Dignifiedquire commented:
@theghostbel ping

theghostbel commented:
pong, one month routetrip :) not bad
will try to do this tomorrow, not a big fan console git, prefer hg with it's GUI :)

theghostbel commented:
I signed it!

theghostbel commented:
Any plan to merge this PR?
